### DISCUSSION TITLE
How to control IR and reboot cam via web?

### DISCUSSION THREAD

**Discussion by LeoI5** [Arkady23/V380-Web#57] [General]
I use V380 cam with Home Assistant anв want to control it.
Is it possible to control cam via rest request via cam web server?

**Comment by Arkady23** [comment]
The option is available in the web interface settings. Or did I not understand your question?

**Reply by LeoI5** [reply]
I mean Restful API for Home Assistant [RESTful Switch](https://www.home-assistant.io/integrations/switch.rest/)

**Reply by Anton-Ka** [reply]
> The option is available in the web interface settings.

My camera server sometimes stops responding.
I would also like to be able to reboot the camera remotely, for example by sending a command or request from the home server.

Сервер моей камеры иногда перестает отвечать.
Я бы тоже хотел иметь возможность перезагрузить камеру удаленно, например отправив команду или запрос с домашнего сервера.

**Reply by Anton-Ka** [reply]
Я нашел такую [ссылку](https://stackoverflow.com/questions/30613150/powershell-script-to-click-a-javascript-hyperlink) с похожим вопросом, но моих знаний не хватает, чтобы сделать из неё полноценную команду, которую я бы мог привязать к кнопке в Home assistant.

**Reply by LeoI5** [reply]
идешь в chatGPT b просишь оформить в интеграцию или в сенсор ХА.
